assets/Scripts/Settings/AudioHandeler.cs

```diff
@@ -0,0 +1,33 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+public class AudioHandler : MonoBehaviour
```

Comment:
please change the file name "AudioHandler", currently it is "AudioHandeler"

Reply:
removed this entirely, was an old script